.github/workflows/release.yml

```diff
@@ -222,6 +222,27 @@ jobs:
     needs: [ build, version-tag ]
     if: needs.version-tag.outputs.append_notes == 'true'
     steps:
+      - name: Get Release
```

Comment:
- RESTful architecture guidelines recommend using appropriate HTTP methods and URIs. Ensure the endpoints are aligned with REST principles.